src/main/java/org/togetherjava/discordbot/commands/CommandListener.java

```diff
@@ -12,15 +12,19 @@
 import net.dv8tion.jda.api.JDA;
 import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.togetherjava.discordbot.commands.CommandContext.JdaRequestContext;
 import org.togetherjava.discordbot.config.TjBotConfig;
 import org.togetherjava.discordbot.util.Messages;
 
+import java.lang.invoke.MethodHandles;
+
 /**
  * A command listener for JDA. It is responsible for listening to messages an invoking commands.
  */
 public class CommandListener extends ListenerAdapter {
-
+  private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
See comment on `MessageHandles` in `BotMain`. I believe this class should have a logger but make sure to keep formatting the same, as you accidentally deleted the empty line put between the first variable and the class declaration line.